src/hammer-tech/schema.json

```diff
@@ -149,6 +152,9 @@
     "name" : {
       "type" : "string"
     },
+    "gds map file" : {
```

Comment:
I don't think it makes sense to have more than one map. It definitely won't vary by RC corner. I think it would only vary, possibly, by who exactly builds your chip. for example some intermediary might run a check that certain layers are or are not present.